### PR TITLE
Feature tmc#88 scl eventually

### PR DESCRIPTION
**Pull request by rphh**

### CODE REVIEW

File: tests/promela/test_model.py [hunk 1]
```diff
@@ -98,6 +98,47 @@ def test_do():
     generate_and_verify(model, "do.pml")
 
 
+def test_atomic_alternative():
```
Comment (Lurkerpas) [inline tests/promela/test_model.py:101]: No test for eventually?

Comment (rphh) [reply tests/promela/test_model.py:101]: I can add some integration tests for this. I don't want to stick to a "valid" form of never claim, I anticipate many changes in this area, so integration test (executing spin on the model) should good.

File: tests/stop_condition_eventually/Makefile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	cp ${RES_DIR}/dataview.pml ${TEST_DIR}/dataview.pml
+	cp ${RES_DIR}/system.pml ${TEST_DIR}/system.pml
+	cp ${RES_DIR}/env_inlines.pml ${TEST_DIR}/env_inlines.pml
+	cd ${TEST_DIR} && \
+	${SPIN} -a -m system.pml
+	$(CC) -o ${TEST_DIR}/pan ${TEST_DIR}/pan.c $(CFLAGS) -DNFAIR=3
+	cd ${TEST_DIR} && \
+	./pan -a -f -n -m1000000 > system.output
+	grep -q "pan:1: acceptance cycle" ${TEST_DIR}/system.output
+	grep -q "errors: 1" ${TEST_DIR}/system.output
```
Comment (Lurkerpas) [inline tests/stop_condition_eventually/Makefile:53]: Why error is expected?

File: tests/stop_condition_eventually/Makefile [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+# glue code generated by Kazoo), they are wrapped inside "host" Promela
+# files
+
+test_stop_condition_eventually:
+	echo "Test stop_condition_eventually"
+	${CREATE_EMPTY_TEST_DIR}
+	${SDL2PROMELA} -o ${TEST_DIR}/controller.pml ${RES_DIR}/controller/system_structure.pr ${RES_DIR}/controller/controller.pr
+	${SDL2PROMELA} -o ${TEST_DIR}/actuator.pml ${RES_DIR}/actuator/system_structure.pr ${RES_DIR}/actuator/actuator.pr
+	${SDL2PROMELA} -o ${TEST_DIR}/scl.pml --scl ${RES_DIR}/scl.txt
+	cp ${RES_DIR}/dataview.pml ${TEST_DIR}/dataview.pml
```
Comment (Lurkerpas) [inline tests/stop_condition_eventually/Makefile:44]: Also, I remember that my original comment was about eventually.... but this MR also introduces filter out. Is it tested?

File: tests/stop_condition_eventually/Makefile (hunk 2 above)
Comment (rphh) [reply tests/stop_condition_eventually/Makefile:53]: I don't understand your concerns. If model contains an error, then the model checker shall find it. It is normal test scenario IMO.

File: tests/stop_condition_eventually/Makefile (hunk 3 above)
Comment (rphh) [reply tests/stop_condition_eventually/Makefile:44]: No, It wasn't. I've added a test for filter_out.

File: tests/promela/test_model.py (hunk 1 above)
Comment (Lurkerpas) [reply tests/promela/test_model.py:101]: Integration tests added

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**